bigbluebutton-html5/imports/ui/components/loader/component.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
oh i thought he actually meant we import component without react everywhere else.